RealmSwift/Tests/ObjectCreationTests.swift

```diff
@@ -441,7 +441,7 @@ class ObjectCreationTests: TestCase {
         Realm().commitWrite()
         switch type {
             case .Bool:     return ["invalid", 2 as Int, 1.1 as Float, 11.1 as Double]
-            case .Int:      return ["invalid", true, false, 1.1 as Float, 11.1 as Double]
+            case .Int:      return ["invalid", 1.1 as Float, 11.1 as Double]
```

Comment:
So because of your changes to `nsnumber_is_like_integer`, Boolean values can now be assigned to `Int` properties? Why?